ci-operator/jobs/openshift/openshift-ansible/openshift-openshift-ansible-presubmits.yaml

```diff
@@ -466,7 +466,7 @@ presubmits:
         - name: TEST_FOCUS
           value: Suite:openshift/conformance/parallel
         - name: JOB_NAME_SAFE
-          value: e2e-gcp
+          value: e2e-gcp-master-major-upgrade
```

Comment:
If you change this you also need to change the job volume mounts.

Reply:
And you should change this (Otherwise a gcp and gcp upgrade job can't run at the same time).

Please see e2e-gcp-crio for an example of how you keep JOB_NAME_SAFE and mount path in sync.

Reply:
Right, changed this to `e2e-gcp-major-upgrade-master`, so that later on we could create 3.9 -> 3.10 upgrade job with `e2e-gcp-major-upgrade-3.11` and so forth.

Updated mountpoints